src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -22,6 +22,7 @@
         "symfony/class-loader": "~3.2",
         "symfony/dependency-injection": "~3.2",
         "symfony/config": "~2.8|~3.0",
+        "symfony/console": "~3.2",
```

Comment:
This can be removed. Because of the class_exists call, if the installed component is an older version or isn't installed it will still work. Maybe should you add a another test to not load the deprecated pass if the component is not installed at all (to prevent throwing an illegitimate deprecation warning).